app/src/main/java/io/pslab/fragment/AboutUsFragment.java

```diff
@@ -47,7 +47,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                 .addWebsite("https://goo.gl/forms/sHlmRAPFmzcGQ27u2", getString(R.string.nav_report))
                 .addItem(new Element(getString(R.string.version), R.drawable.ic_widgets_black_24dp))
                 .addItem(new Element(getString(R.string.flavor), R.drawable.ic_android_black_24dp))
-                .setDescription(getString(R.string.about_us_description))
+                .setDescription(getString(R.string.about_us_description)+getString(R.string.about_fossasia_description))
```

Comment:
Why not creating the complete string in `strings.xml` without concatenating here?

Reply:
I've done it this way now. I though it would be easier for future reference with two strings